### ISSUE TITLE
Number of checkboxes is not announced when user lands on the first checkbox in the group

### ISSUE THREAD

**Issue by gurooprasad** [nvaccess/nvda#11295]
### Steps to reproduce:
Open the URL: https://www.accessibility-developer-guide.com/examples/forms/good-example/_examples/generally-good-form/
Navigate through the checkbox and radio group with NVDA on using tab key navigation. Observe the screen reader's spoken feedback when the first checkbox or radio button is focused.

### Actual behavior:
Number of items is only announced fo the radio button.

### Expected behavior:
For checkbox or radio button number of items in the group should be announced along with other relevant info related to the respective element.

**Comment by Adriani90** [comment]
This is reproducible in Firefox 77 and also in Chrome 83. I am not sure if this is exposed by the browsers for checkboxes at all.
cc: @jcsteh, @ObjectInSpace

**Comment by jcsteh** [comment]
Even though you can put them in a group, check boxes are not considered to be positional elements by any accessibility spec or framework, so this information is not exposed. Thus, this is expected behaviour.

**Comment by gurooprasad** [comment]
I think that's the point. The bug can be converted to feature request to communicate that info.

**Comment by jcsteh** [comment]
It can't, though. Browsers don't expose it, so screen readers can't get at
it. And browsers don't expose it because the spec says they shouldn't. If
you wanted this to change, that would have to be addressed at the spec
level.

I'm also not convinced there is any value to users. It's important for
users to understand how many radio buttons there are because they have to
choose one from a finite set of options. Same for list boxes. In contrast,
even though they can be grouped, check boxes are completely separate
entities; you can put them wherever you like. If you want them to be a
"list", I'd say a listbox with checkable options is a more appropriate
choice.

**Comment by gurooprasad** [comment]
I have seen list of checkboxes where the list is long. Providing this info helps users to make the navigation bit predictive.
if it doesn't add any value then number of items shouldn't be announced for any type of list or items in a group. Thanks for your feedback. Would like to see more opinions from more people, as I always feel individual’s assumptions can be biased sometimes.

**Comment by jcsteh** [comment]
Fair enough. If this is something you feel strongly about, I'd suggest you
take this up with the ARIA spec group. It's not something a screen reader
can do without changes to the spec.
